assets/airports/efhk.json

```diff
@@ -0,0 +1,1565 @@
+{
+    "radio": {
+        "twr": "Helsinki Tower",
+        "app": "Helsinki Radar",
+        "dep": "Helsinki Radar"
+    },
+    "icao": "EFHK",
+    "iata": "HEL",
+    "magnetic_north": 8.4,
+    "ctr_radius": 60,
+    "ctr_ceiling": 28500,
+    "initial_alt": 4000,
+    "position": ["N60d19m02", "E24d57m48", "180ft"],
+    "rr_radius_nm": 5,
+    "rr_center": ["N60d19m02", "E24d57m48"],
+    "has_terrain": true,
+    "wind": {
+        "angle": 240,
+        "speed": 8
+    },
+    "arrivalRunway": "22L",
+    "departureRunway": "22R",
+    "airspace": [
+        {
+            "floor": 0,
+            "ceiling": 285,
+            "airspace_class": "B",
```

Comment:
The first item should include all of it. You may take a look into the official docs:

https://www.ais.fi/ais/aip/aipcharts/efhk/EF_AD_2_EFHK_ARC.pdf
https://www.ais.fi/ais/aip/aipcharts/efhk/EF_AD_2_EFHK_VAC.pdf

But there's also information about the airspace restructuring that contains an easy to comprehend sideways view about the airspace shelving:

http://www.urheiluilmailuopisto.fi/assets/files/materiaali/ilmatilaseminaari/Ilmatilauudistus%20esittely%20VANTAA%2020150310.pdf